pyperformance/cli.py

```diff
@@ -252,6 +252,11 @@ def parse_entry(o, s):
         # Filter out any benchmarks that can't be run on the Python version we're running
         if this_python_version in bench.python:
             selected.append(bench)
+        else:
+            message = (f"Benchmark: {bench.spec.name} is being skipped because "
+                       f"python version required is {bench.python} "
+                       f"but found {this_python_version}")
+            logging.info(message)
```

Comment:
Use lazy logging: rather than building an f-string which will always be evaluated, use percent formatting for so it's only evaluated if run.

```python
logging.info("Benchmark: %s is...", bench.spec.name)
```